src/consumerIncentives/slice.ts

```diff
@@ -70,10 +70,18 @@ const slice = createSlice({
     builder.addCase(
       AppActions.UPDATE_REMOTE_CONFIG_VALUES,
       (state, action: UpdateConfigValuesAction) => {
-        state.superchargeV2Enabled = action.configValues.superchargeV2Enabled
-        state.superchargeRewardContractAddress =
-          action.configValues.superchargeRewardContractAddress
-        state.superchargeV1Addresses = action.configValues.superchargeV1Addresses
+        // state.superchargeV2Enabled = action.configValues.superchargeV2Enabled
+        // state.superchargeRewardContractAddress =
+        //   action.configValues.superchargeRewardContractAddress
+        // state.superchargeV1Addresses = action.configValues.superchargeV1Addresses
+        state.superchargeV2Enabled = true
+        state.superchargeRewardContractAddress = '0x530cf57b70c22C0437bd77aDeBd79Fe58c42dC2A'
+        state.superchargeV1Addresses = [
+          '0x15d6394bA9f3A0095a60e959Fc02598484d46a05',
+          '0x958cF6E205AF70628D4b83A16a92F3DD92C6bB50',
+          '0x285cE0Aaea13221821Cfac370904E3C617d0b82C',
+          '0x06821F2b98F6709aC919b32cAF03119566e6f3c5',
```

Comment:
change these contract addresses to the ones you need! these particular values are for cEUR only